bitnami/mariadb/values.yaml

```diff
@@ -710,24 +716,30 @@ secondary:
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod
   ## @param secondary.podSecurityContext.enabled Enable security context for MariaDB secondary pods
   ## @param secondary.podSecurityContext.fsGroup Group ID for the mounted volumes' filesystem
+  ## @param secondary.podSecurityContext.seccompProfile.type Set container's Security Context seccomp profile
   ##
   podSecurityContext:
     enabled: true
     fsGroup: 1001
+    seccompProfile:
+      type: "RuntimeDefault"
```

Comment:
`seccompProfile` is not set in `primary.podSecurityContext`. Sorry I didn't see it before